client/js/entities/worldItem.js

```diff
@@ -0,0 +1,85 @@
+define(function(require) {
+
+var THREE = require("THREE");
+var EntityBar = require("meshes/entityBar");
+var HitboxMesh = require("meshes/hitboxMesh");
+var Box = require("physics/box");
+
+return function WorldItem(worldItemState) {
+	var self = this;
+	var hitboxMesh;
+
+	self.kind = function() {
+		return worldItemState.itemKind;
+	};
+	self.pos = function() {
+		return worldItemState.entityState.body.pos.clone();
+	};
+	self.look = function () {
+		return worldItemState.entityState.body.dir.clone();
+	};
+	self.halfExtents = function() {
+		return worldItemState.entityState.body.halfExtents;
+	};
+	self.box = function() {
+		return new Box(self.halfExtents(),
+			worldItemState.entityState.body.centerOffset).setPosition(self.pos());
+	};
+	self.contains = function(x, y, z) {
+		return self.box().contains(x, y, z);
+	};
+
+	var mesh = new THREE.Object3D();
+	self.addTo = function(scene) {
+		scene.add(mesh);
+	};
+	self.removeFrom = function(scene) {
+		scene.remove(mesh);
+	};
```

Comment:
Self properties and update functions are definitely different. If we end up with a third class like this, I'll refactor them to share more logic from a base class thing.